cluster-init/ocp-4.4/operator-crw-cr.yaml

```diff
@@ -8,6 +8,7 @@ spec:
     cheImageTag: ''
     cheFlavor: codeready
     devfileRegistryImage: ''
+    externalPluginRegistry: true # add it for CRW 2.3 on OCP 4.5 at least
```

Comment:
Does this new attribute is related to an upgrade of the Custom Resource version ? Is it still compatible with CRW on OCP 4.4 ?

Reply:
I will update the comment. It is not related OCP version.